ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-openshift-logging.yaml

```diff
@@ -0,0 +1,91 @@
+base_images:
+  ansible:
+    name: "4.12"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.12"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  hypershift-operator:
+    name: "4.12"
+    namespace: ocp
+    tag: hypershift-operator
+  openstack-installer:
+    name: "4.12"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.12"
+  tests-private-ui:
+    name: tests-private-ui
+    namespace: ci
+    tag: "4.12"
+  tools:
+    name: "4.12"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.12"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  latest:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.12"
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: azure-ipi-fips-p3-f1-openshift-logging
+  cron: 0 1 * * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
```

Comment:
@IshwarKanse Do we need to set `COMPUTE_NODE_TYPE` here? Does the default worker node type meet our test requirements?

Reply:
The default is Standard_D4s_v3 which should be fine, I'll add it in our config for easy reference along with SIZE_VARIANT: "large" 

```
  - name: SIZE_VARIANT
    default: ""
    documentation: |-
      The size of the cluster in one of our supported t-shirt values that is standard across all CI environments.

      The sizes are:
      * "" (default) - 4 vCPU, 16GB control plane nodes, default workers
      * "compact" - 8 vCPU, 32GB control plane nodes, no workers
      * "large" - 16 vCPU, 64GB+ control plane nodes, default workers, suitable for clusters up to 250 nodes
      * "xlarge" - 32 vCPU, 128GB+ control plane nodes, default workers, suitable for clusters up to 1000 nodes

      These sizes are roughly consistent across all cloud providers, but we may not be able to instantiate some sizes
      in some regions or accounts due to quota issues.

  - name: COMPUTE_NODE_TYPE
    default: 'Standard_D4s_v3'
    documentation: |-
      The instance type to use for compute nodes
```
https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/conf/azure/ipi-conf-azure-ref.yaml